### DISCUSSION TITLE
Bun support?

### DISCUSSION THREAD

**Discussion by TheBearodactyl** [neoclide/coc.nvim#5202] [Ideas]
**Is your feature request related to a problem? Please describe.**
Node is slow.

**Describe the solution you'd like**
Bun support.

**Describe alternatives you've considered**
[boa](https://github.com/boa-dev/boa), but that probably wouldn't work in production considering it's not stable yet

**Additional context**
none really, i just wanna be able to use bun

**Comment by fannheyward** [comment]
No plan for now.

Someone has already made some test with Bun https://github.com/neoclide/coc.nvim/discussions/4767

**Comment by TheBearodactyl** [comment]
oh didn't see that
